manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+
+  "manifest_version": 2,
+  "name": "Shield Study WebExtension Experiment",
+  "version": "1.0",
+
+  "description": "A Shield Study WebExtension Experiment to change a preference in Firefox.",
+
+  "applications": {
+    "gecko": {
+      "id": "shield-study-webextension-experiment@shield.mozilla.org",
+      "strict_min_version": "59.0"
+    }
+  },
+
+  "background": {
+  "scripts": ["background.js"]
```

Comment:
nit: indenting looks a bit off here. But super small micro-nit, since I can run everything through **prettier** (<kbd>npm run format</kbd>) and autoformat and submit a PR.